src/utils/slugizeProductArea.test.ts

```diff
@@ -0,0 +1,21 @@
+import { slugizeProductArea } from './slugizeProductArea';
+
+describe('slugizeProductArea', function () {
+  describe('slugize tests', function () {
+    it('behaves as expected', async function () {
+      const cases = [
```

Comment:
Feels like someone needs https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Oooo, thanks! I searched the docs for "parameters" and came up empty. Is `test` the same as `it`? 🤔

Reply:
Yup, they are the same 😊

Reply:
#447 💃